### PR TITLE
🌐 Add Portuguese translation for `docs/deployment/versions.md`

### PR DESCRIPTION
**Pull request by stlucasgarcia**
Hey 👋🏻

This PR adds the Portuguese translation of `deployment/versions.md`. I've also added it to `mkdocs.yml`. 
The translation was a bit tricky because of word repetition, I tried my best to keep it the closest to the original. If you think I did something wrong or that a section doesn't make sense, just give me a review and I will fix it asap 🚀 

I also took the liberty of changing the `https://semver.org/` link to the Portuguese translation `https://semver.org/lang/pt-BR/`

### CODE REVIEW

File: docs/pt/docs/deployment/versions.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Novos recursos são adicionados com frequência, bugs são corrigidos regularmente e o código está sempre melhorando.
+
+Esse é o motivo das versões atuais estarem em `0.x.x`, significando que em cada versão pode haver mudanças significativas, tudo isso seguindo as <a href="https://semver.org/lang/pt-BR/" class="external-link" target="_blank">convenções de controle de versão semântica.</a>
+
+Já é possível criar aplicativos de produção com **FastAPI** (e provavelmente você já faz isso há algum tempo), apenas precisando ter certeza de usar uma versão que funcione corretamente com o resto do seu código.
+
+## Fixe a sua versão de `fastapi`
+
+A primeira coisa que você deve fazer é "fixar" a versão do **FastAPI** que você está utilizando na versão específica mais recente, na qual você sabe que funciona corretamente para o seu aplicativo.
```
Comment (Kludex) [inline docs/pt/docs/deployment/versions.md:13]: Maybe?
```suggestion
A primeira coisa que você deve fazer é "fixar" a versão do **FastAPI** que você está utilizando na mais atual, na qual você sabe que funciona corretamente para o seu aplicativo.
```

File: docs/pt/docs/deployment/versions.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+Você pode ver as versões disponíveis (por exemplo, para verificar qual é a versão atual) em [Release Notes](../release-notes.md){.internal-link target=\_blank}.
+
+## Sobre versões
+
+Seguindo as convenções de controle de versão semântica, qualquer versão abaixo de `1.0.0` pode adicionar mudanças significativas.
+
+FastAPI também segue a convenção de que qualquer alteração de versão "PATCH" é para correção de bugs e alterações não significativas.
+
+!!! tip
```
Comment (Kludex) [inline docs/pt/docs/deployment/versions.md:45]:
```suggestion
!!! tip "Dica"
```

File: docs/pt/docs/deployment/versions.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+Logo, você deveria conseguir fixar a versão, como:
+
+```txt
+fastapi>=0.45.0,<0.46.0
+```
+
+Mudanças significativas e novos recursos são adicionados em versões "MINOR".
+
+!!! tip
```
Comment (Kludex) [inline docs/pt/docs/deployment/versions.md:56]:
```suggestion
!!! tip "Dica"
```

File: docs/pt/docs/deployment/versions.md (hunk 1 above)
Comment (lsglucas) [reply docs/pt/docs/deployment/versions.md:13]: Sounds good!

**Review by Kludex — APPROVED** [review]

**Review by oandersonmagalhaes — APPROVED** [review]

**Comment by tiangolo** [comment]
Awesome, thanks for the contribution @lsglucas! :cake: :coffee: 

And thanks for the reviews @Kludex and @oandersonmagalhaes! :rocket: :nerd_face:

**Pull request merged**